Prisma

nickothan opened the pull request.

nickothan committed:
objetos 1

nickothan committed:
LAS-9: implementado modelos( error en relacion emergencyContact y BasicService.)

nickothan committed:
LAS-9: integrando prisma client

nickothan committed:
LAS-9: modelo de property.

nickothan committed:
LAS-9: manager property.

nickothan committed:
LAS-9: actions and mutations from property.

nickothan committed:
LAS-9: manager and actions from property.

revione committed:
LAS-9: (feat) console the properties in dashboard

revione committed:
LAS-9: use properties

revione committed:
LAS-9: Add property page

revione committed:
LAS-9: remove package lock

nickothan committed:
LAS-9: quit coments from manager property.

nickothan committed:
LAS-10: Model Address property.

nickothan committed:
LAS-10: models from information property.

nickothan committed:
LAS-10: actions and mutations and manager from information property.

nickothan committed:
LAS-10: actions and mutations and manager from features.

nickothan committed:
LAS-10: update actions from information property.

nickothan committed:
LAS-10: manager and actions from interior model.

nickothan committed:
LAS-10: actions and mutations from exterior information.

nickothan committed:
LAS-10: manager and actions from address.

nickothan committed:
LAS-10: read information property.

nickothan committed:
LAS-10: model, manager, actions and read from information properties.

nickothan committed:
LAS-10: style from information property page.

nickothan committed:
LAS-11: Model legal.

nickothan committed:
LAS-11: Manager feom legal info.

nickothan committed:
LAS-11: actions and mutations from legal info.

nickothan committed:
LAS-11: read legal info on page.

nickothan committed:
LAS-12: add model tentants.

nickothan committed:
LAS-12: add manager to tentant.

nickothan committed:
LAS-12: actions and mutations from tentants

nickothan committed:
LAS-12: AdditionalResident manager.

nickothan committed:
LAS-12: AdditionalResident actions and mutations.

nickothan committed:
LAS-12:  paymentTenant manager.

nickothan committed:
LAS-12:  paymentTenant actions and mutations.

nickothan committed:
LAS-12: read tenants.

nickothan committed:
LAS-13: Model from Insurance

nickothan committed:
LAS-13: manager from Insurance

nickothan committed:
LAS-13: manager from Insurance

nickothan committed:
LAS-13: manager from insurances.

nickothan committed:
LAS-13: actions adn mutations from insurances.

nickothan committed:
LAS-13: read insurances in page.

nickothan committed:
LAS-13:quit coment.

nickothan committed:
LAS-14: BasicService models.

nickothan committed:
LAS-14: BasicService manager.

nickothan committed:
LAS-14: BasicService actions and mutations.

nickothan committed:
LAS-14: billingDetails manager.

nickothan committed:
LAS-14: billingDetails actions and mutations..

nickothan committed:
LAS-14: servicePayment manager.

nickothan committed:
LAS-14: servicePayment actions and mutations..

nickothan committed:
LAS-14: consumptionDetails manager.

nickothan committed:
LAS-14: consumptionDetails actions and mutations.

nickothan committed:
LAS-14: meterReading manager.

nickothan committed:
LAS-14: meterReading actions and mutations.

nickothan committed:
LAS-14: additionalService manager.

nickothan committed:
LAS-14: additionalService actions and mutations.

nickothan committed:
LAS-14: emergencyContact manager.

nickothan committed:
LAS-14: emergencyContact actions and mutations.

nickothan committed:
LAS-14: specificConditions manager.

nickothan committed:
LAS-14: specificConditions actions and mutations.

nickothan committed:
LAS-14: incident manager.

nickothan committed:
LAS-14: incident actions and mutations.

revione committed:
LAS-10: (feat) Information

- Update schema with uuid
- Use slag for the property
- get the property information
- show the property information
- show the address in the dashboard

revione committed:
LAS-11: (feat) Legal

- update the schema names
- use the id to get the property legal information
- print the information in the page
- create the new methods insted pass the properties down

nickothan committed:
LAS-13: manager and actions from insurance in proparty files.

nickothan committed:
LAS-13: read insurances from property.

revione commented on src/actions/property/actions_and_mutations.ts:
```diff
@@ -102,3 +102,13 @@ export async function getPropertyLegalByPropertyId({ id }: { id: string }) {
   }
 }
+
+// PropertyInsurance
```
No comment required

revione commented:
remove the other file that will no more used _Insurance_

nickothan committed:
LAS-13: withput coments in page actions property.

nickothan committed:
LAS-12: manager and actions from tenant in proparty files.

nickothan committed:
add property to name models.

nickothan committed:
LAS-12: read from tenants property and update models froms prisma.

nickothan committed:
LAS-12: DRAFT.

revione committed:
LAS-12 - LAS-13: (feat) Tenat - Insurence

- update the schema
- update the seeds
- clean the unused files

nickothan committed:
LAS-14: manager from services in property.

nickothan committed:
LAS-14: actions from services in property.

nickothan committed:
LAS-14: update prisma for services.

nickothan committed:
LAS-14: seeds from services and add in property seeds.

nickothan committed:
LAS-14: read the services in page services.

nickothan committed:
LAS-14: manager from preventive maintenance in property.

nickothan committed:
LAS-15: actions from preventive maintenance in property.

nickothan committed:
LAS-15: seeds from preventive maintenance and add in property seeds.

nickothan committed:
LAS-15: read  preventive maintenance and template with navigation.

nickothan committed:
LAS-15: update module corrective maintenance from prisma.

nickothan committed:
LAS-14: manager from corrective maintenance in property.

nickothan committed:
LAS-15: actions from corrective maintenance in property.

nickothan committed:
LAS-15: seeds from corrective maintenance and add in property seeds.

nickothan committed:
LAS-15: read  corrective maintenance and template with navigation.

nickothan committed:
LAS-15: manager from request maintenance in property.

nickothan committed:
LAS-15: actions from request maintenance in property.

nickothan committed:
LAS-15: seeds from request maintenance and add in property seeds.

nickothan committed:
LAS-15: read  request maintenance and template with navigation.

nickothan committed:
LAS-15: manager from inspectionHistory maintenance in property.

nickothan committed:
LAS-15: actions from inspectionHistory maintenance in property.

nickothan committed:
LAS-15: seeds from inspectionHistory maintenance and add in property seeds.

nickothan committed:
LAS-15: read  inspectionHistory maintenance and template with navigation.

nickothan committed:
LAS-15: manager from maintenanceNotes maintenance in property.

nickothan committed:
LAS-15: actions from maintenanceNotes maintenance in property.

nickothan committed:
LAS-15: seeds from maintenanceNotes maintenance and add in property seeds.

nickothan committed:
LAS-15: read  maintenanceNotes maintenance and template with navigation.

revione committed:
LAS-13 Insurances

revione committed:
LAS-10: Back Button

nickothan committed:
LAS-14 :services page one yo many.

nickothan committed:
LAS-14: payment services seeds

revione committed:
LAS-14: services with paymentService

nickothan committed:
LAS-14: services pages.

nickothan committed:
delete service.

revione committed:
LAS-14: Summarize all the lines as much as possible.

nickothan committed:
LAS-15: models of maintenances.

nickothan committed:
LAS-15: model inspections history deleted.

nickothan committed:
LAS-15: seeds from maintenances.

nickothan committed:
LAS-15: implement seeds and import in properties to create in DB.

nickothan committed:
LAS-15: delete seeds and modify names model.

nickothan committed:
LAS-15: update name.

nickothan committed:
LAS-15: manager and actions from maintenance property.

nickothan committed:
LAS-15: read maintenance info and maintenances.

nickothan committed:
LAS-15: read equipamant and maintenances cards from maintenance property.

nickothan committed:
LAS-15: style cards.

nickothan committed:
LAS-14: models services and create seeds and create from db.

nickothan committed:
LAS-14: read payment and insidents services.

nickothan committed:
LAS-13: read some info about insurances.

nickothan committed:
LAS-20: navigation to property from dashboard.

revione committed:
LAS-26: (feat) Types and order for information property

nickothan committed:
LAS-17: Model from settings.

nickothan committed:
LAS-17: Delete ids to model from settings.

revione committed:
LAS-27: (feat) Legal - update styles

revione committed:
LAS-27: (feat) Pages styles

revione committed:
LAS-27: (feat) Use the modules as separate files.

revione committed:
LAS-15:  three key factors change the paradigm.

We are set to change the paradigm by introducing a new implementation and theory behind the labels. These labels, traditionally used as relevant information, will now depend on three key factors:

Architecture – Represents the property’s structural attributes.

Equipment – Pertains to features and amenities associated with the property.

Economy – Relates to economic activities connected to the property.

Based on these factors, both the labels and models will require a comprehensive redesign.

revione committed:
LAS-28: Create the seeds for the new structure

revione committed:
LAS-24: update the adress show in the card

revione committed:
LAS-29: Update the nav labels

revione committed:
LAS-30: Clean design

revione committed:
LAS-30: Economy

revione committed:
LAS-30: Equipments

revione committed:
LAS-30: Architectures
